.size-snapshot.json

```diff
@@ -1,45 +1,45 @@
 {
   "index.js": {
-    "bundled": 18330,
-    "minified": 7401,
-    "gzipped": 2776,
+    "bundled": 184,
+    "minified": 160,
+    "gzipped": 94,
```

Comment:
Wondering why this changes.

Reply:
Haha, I have no idea. Makes me feel impressive for `shx` even more

Reply:
I don't know either. Would you just revert the change to this file? It avoids to have a revert commit in the master branch.